### PR TITLE
[DEV-3402] HubSpot Updates

### PR DESCRIPTION
**Pull request by jayden-chan**
## Description of the change

PR for any HubSpot schema updates

* Fixed typos in GA `user.created` webhook schema
* Add integration managed user events

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: DEV-3402
- Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

**Comment by DerekSiemens** [comment]
@slisaasquatch or @jayden-chan this needs a quick changelog update

**Review by johanventer — COMMENTED** [review]
Looks good, needs final publish before approval and merging

File: CHANGELOG.md
Comment (DerekSiemens) [inline CHANGELOG.md:8]: Please update to 2023-04-05 before launch today

Comment (slisaasquatch) [reply CHANGELOG.md:8]: I just saw this. Too late. I deployed it already.

Comment (DerekSiemens) [inline CHANGELOG.md:8]:
```suggestion
## [1.25.0] - 2023-04-05
```

**Review by DerekSiemens — APPROVED** [review]

**Pull request merged**
